spec/acceptance/php_spec.rb

```diff
@@ -14,12 +14,8 @@
       packagename = 'php7.4-fpm'
     when %r{ubuntu-18.04}
       packagename = 'php7.2-fpm'
-    when %r{ubuntu-16.04}
```

Comment:
If you're removing it here, perhaps it should also be changed on line 39. That can probably be `when %r{ubuntu-}` to catch all Ubuntu versions. Or `if default[:platform].startwith('ubuntu')` instead of a `case` statement.

Reply:
it simply should not be removed. why removing stuff which is working? there is still ubuntu security support for that version. https://en.wikipedia.org/wiki/Ubuntu#Releases

Reply:
removing 16.04 is perhaps a bit premature, but the EOL date is in 6 weeks so i removed it already to be a bit efficient